src/python_service_template/app.py

```
@@ -29,99 +44,30 @@
 )
 app.include_router(coffee_router)
 app.include_router(health_router)
-
-
-log = structlog.get_logger("exception")
+instrumentator = Instrumentator().instrument(app)
+app.state.instrumentator = instrumentator
 
 
 @app.exception_handler(Exception)
 async def global_exception_handler(request: Request, exc: Exception) -> JSONResponse:
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)